lms/templates/problem.html

```diff
@@ -58,27 +62,35 @@ <h3 class="hd hd-2 problem-header" id="${ id }-problem-title" aria-describedby="
             <%include file="problem_notifications.html" args="
                 notification_type='success',
                 notification_icon='check',
-                notification_message=answer_notification_message,
-                hint_notification=hint_notification"
+                notification_name='submit',
+                notification_message= answer_notification_message"
             />
         % endif
         % if 'incorrect' == answer_notification_type:
             <%include file="problem_notifications.html" args="
                 notification_type='error',
                 notification_icon='close',
-                notification_message=answer_notification_message,
-                hint_notification=hint_notification"
+                notification_name='submit',
+                notification_message= answer_notification_message"
             />
         % endif
         % if 'partially-correct' == answer_notification_type:
             <%include file="problem_notifications.html" args="
                 notification_type='success',
                 notification_icon='asterisk',
-                notification_message=answer_notification_message,
-                hint_notification=hint_notification"
+                notification_name='submit',
+                notification_message= answer_notification_message"
             />
         % endif
     % endif
+    % if save_notification_message:
+      <%include file="problem_notifications.html" args="
+        notification_type='warning',
+        notification_icon='save',
+        notification_name='save',
+        notification_message=_(save_notification_message)"
```

Comment:
The translation shouldn't be happening here. The scraping and translation happens where the string is defined.